src/flstring.h

```diff
@@ -52,10 +52,6 @@
  * Some of these functions are also defined in ISO C99...
  */
 
-#  if defined(_MSC_VER)
-#    define strdup _strdup
-#  endif /* _MSC_VER */
-
```

Comment:
Figured this should be removed, so we get errors/warnings again if someone uses strdup() in VS.

Reply:
:+1:

Reply:
:+1: